### DISCUSSION TITLE
How can I use sqlc if I don't know how to write sql files?

### DISCUSSION THREAD

**Discussion by frederikhors** [sqlc-dev/sqlc#1033] [Q&A]
I'm using [go-pg](https://pg.uptrace.dev/) right now.

I wanna try `sqlc` but I'm too inexperienced to define PG tables properly.

For example I don't know what is the right choice for `ID` columns and other definitions which instead using `Gorm` or `go-pg` come to me "FREE".

**What do you recommend to those like me in this situation?**

**Comment by mvrhov** [comment]
That you borrow a book an learn SQL

**Comment by kyleconroy** [comment]
sqlc requires a good understanding of SQL. There's no shame in using other Go ORMs that require less SQL knowledge. If you're just getting into SQL, this is a good place to start: https://gumroad.com/l/sql-zine.

**Comment by app-o-matix** [comment]
You might also check out [dbdiagram.io](url) as a step toward learning sql. It's sql-ish and generates sql that can be exported for Postgres, MySql, or Sql Server.
